examples/preprocessing/plot_artifact_detection.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+=============================================
+Detect artifacts
+=============================================
+
+Detects bad channels
+
+Annotates periods where there is excessive movement and calculates a new
+device to head transformation matrix
+
+Annotates segments contaminated with muscle artifacts
+
+"""
+# Authors: Adonay Nunes <adonay.s.nunes@gmail.com>
```

Comment:
since the code was originally adopted from @bloyl maybe you want to add him as one of the authors?

Reply:
Sure!